### PR TITLE
immortal: update to 0.24.4.

### PR DESCRIPTION
**Pull request by tibequadorian**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/immortal/template [hunk 1]
```diff
@@ -13,7 +13,8 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://github.com/immortal/immortal"
 distfiles="https://github.com/immortal/immortal/archive/${version}.tar.gz"
-checksum=e31d5afb9028fb5047b5a2cc5f96c844f6480d600643a12075550f497e65f5cb
+checksum=a343581dbe58fb0faa1c65b233a067820d8d5ecefc9726da5ad3ef979a2a0b08
+make_check=ci-skip # tests pass locally
```
Comment (paper42) [inline srcpkgs/immortal/template:17]: That's not a reason, ci-skip already tells us that the tests pass locally.

Comment (tibequadorian) [reply srcpkgs/immortal/template]: Oh thats right... I've enabled them now. How would you describe this?

Comment (tibequadorian) [reply srcpkgs/immortal/template]: or should i just `make_check=ci-skip` without a comment?

Comment (tibequadorian) [reply srcpkgs/immortal/template]: Updated it now :)

Comment (paper42) [reply srcpkgs/immortal/template]: Tests also fail locally for me, so it shouldn't be ci-skip and it's not caused by being root.

Comment (paper42) [reply srcpkgs/immortal/template]: And now they pass after a clean.

**Pull request merged**